### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by XiangkunYe** [zywan/Deep-Learning#1]
Your work is really complex and elegant, especially the CNN model for five classes or more objects. This model just uses Tensorflow without anything other API, which I guess should be really tough. The Keras model is more readable and easier to understand. But I think there's a common problem for both model that it's easy for them to get overfitting. I think you could try to change some parameters like dropout or just increase the size of train dataset to see if it gets better.

**Comment by XiangkunYe** [comment]
By the way, I'm Xiangkun Ye and my email address is xk1997@bu.edu: )
(hope this make it easier for TA to recognize me)
